src/Illuminate/Support/Stringable.php

```diff
@@ -708,6 +708,25 @@ public function ucfirst()
         return new static(Str::ucfirst($this->value));
     }
 
+    /**
+     * Apply the callback's string changes if the given "value" is false.
+     *
+     * @param  mixed  $value
+     * @param  callable  $callback
+     * @param  callable|null  $default
+     * @return mixed|$this
+     */
+    public function unless($value, $callback, $default = null)
+    {
+        if (! $value) {
+            return $this->when(true, $callback);
+        } elseif ($default) {
+            return $this->when(false, $callback, $default);
+        }
+
+        return $this;
+    }
```

Comment:
Maybe simplify? Similiar `throw_if` and `throw_unless`.

```suggestion
    public function unless($value, $callback, $default = null)
    {
        return $this->when(! $value, $callback, $default);
    }
```

Reply:
Great, thanks